src/ClassicUO.Client/LegionScripting/API.py

```
@@ -299,6 +302,12 @@ def CreateGumpRadioButton(text: str = "", group: int = 0, inactive: int = 0x00D0
 def CreateGumpTextBox(text: str = "", width: int = 200, height: int = 30, multiline: bool = False):
     pass
 
+def CreateGumpTTFLabel(text: str, size: float, color: str = "#FFFFFF", font: str = TrueTypeLoader.EMBEDDED_FONT, aligned: str = "let", maxWidth: int = 0, applyStroke: bool = False):
```

Ruff (0.11.9)

Undefined name `TrueTypeLoader`

(F821)